apps/ui/modules/reservationUnit.ts

```diff
@@ -223,54 +229,56 @@ export const getFuturePricing = (
     : futurePricings[0];
 };
 
+function formatPrice(
+  price: number,
+  trailingZeros: boolean,
+  toCurrency?: boolean
+) {
```

Comment:
Return value: is it supposed to be a `number` or `string`?
Now it's `string | number` which doesn't seem correct.